### DISCUSSION TITLE
DB migration from 0.10.1 to 0.11.0

### DISCUSSION THREAD

**Discussion by p-freire** [milvus-io/milvus#4070] [Q&A and General discussion]
I have a Milvus 0.10.1 server running with close to 100 collections in it and I wanted to upgrade to 0.11.0. So I followed the instructions of "How to migrate data in Milvus?", which are "_Copy the entire db directory of the original Milvus service to the new directory. When restarting the Milvus service, map the copied db directory to the db directory of the Milvus service_". However, when I started Milvus-EM version 0.50 alongside Milvus 0.11.0, my collections were not migrated at all.

Could you please point in the right direction? How can I migrate everything? Is there something I'm missing?

Thank you in advance.

**Comment by gujun720** [comment]
Hi, p-freire,

Unfortunately, since the data management of Milvus 0.11.0 is different from 0.10.0, we could not migrate the data just by copying the db folder.

We will have to import the data again when migrating to 0.11.
